src/SDK/Subscriber/SubscribedEvent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Subscriber;
+
+use function count;
+use OpenTelemetry\SDK\Attributes;
+use OpenTelemetry\SDK\AttributesInterface;
+use OpenTelemetry\SDK\Trace;
+use OpenTelemetry\SDK\Trace\Span;
+
+final class SubscribedEvent implements SubscribedEventInterface
+{
+	
+	private Span $span;
+
+	public function __construct(Span $span)
+	{
+		$this->span = $span;
+	}
+
+	public function dispatch():void
+	{
+		$classname = get_called_class();
```

Comment:
What's the purpose of calling `get_called_class` , when it's a final class?

Reply:
In the implementation the process the user could use to register the Listener with Event is as follows-

- During the instantiation of Span builder class the user will provide the The event class name with which he/she wants to register the listener along-with the Listener class name both in string format.  So to correlate the Listener class with Event class there names have to be known for which "get_called_class" is used.